plugins/etc/httpd/conf.d/pulp_rpm.conf

```diff
@@ -16,19 +16,23 @@
 AddType application/x-pkcs7-crl .crl
 AddType application/x-x509-ca-cert .crt
 
-# -- HTTPS Repositories ---------
-Alias /pulp/repos /var/www/pub/yum/https/repos
+# -- Yum Repositories ---------
+<Location /pulp/repos/>
+  RewriteEngine On
+  RewriteCond %{HTTPS} on
+  RewriteRule (.+/pulp/repos/)(.+) /pulp/content/var/www/pub/yum/https/repos/$2 [DPI]
```

Comment:
Consider dropping the initial .+ here and two lines below, since the Location directive is matching /pulp/repos already. You'll need to switch that $2 to a $1 as well.